aioshelly/rpc_device/device.py

```diff
@@ -304,7 +305,9 @@ async def _init_calls(self) -> None:
         if fetch_status:
             self._status = results.pop(0)
         if fetch_dynamic:
-            self._parse_dynamic_components(results.pop(0))
+            components = results.pop(0)
+            self._parse_dynamic_components(components)
+            await self._retrieve_blutrv_components(components)
```

Comment:
I think we should not call this for all devices that support dynamic components but only for BLU Gateway G3.

Reply:
Good point, will update code